### PR TITLE
alpine: Remove NAME and VERSION variables

### PR DESCRIPTION
**Pull request by castrojo**
As per recommendations in #50

### CODE REVIEW

**Comment by travier** [comment]
You should remove the ENV vars but keep the labels

**Comment by travier** [comment]
You need to move the names and version into the labels as we don't set the env now.

**Comment by castrojo** [comment]
Wow that was embarrassing, ok PTAL.

**Review by travier — APPROVED** [review]

File: alpine/3.16/Containerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 FROM docker.io/library/alpine:3.16
 
-ENV NAME=alpine-toolbox VERSION=3.16
 LABEL com.github.containers.toolbox="true" \
-      name="$NAME" \
-      version="$VERSION" \
+      com.redhat.component="alpine-toolbox" \
```
Comment (travier) [inline alpine/3.16/Containerfile:4]: Hum, I think this only makes sense for RPM based images.

**Review by travier — CHANGES_REQUESTED** [review]

**Comment by travier** [comment]
Sorry one last nit

**Review by travier — APPROVED** [review]

**Pull request merged**